plugins/woocommerce-admin/client/wp-admin-scripts/payment-method-promotions/payment-promotion-row.tsx

```diff
@@ -7,11 +7,14 @@ import { useState, useEffect } from '@wordpress/element';
 import {
 	PLUGINS_STORE_NAME,
 	PAYMENT_GATEWAYS_STORE_NAME,
-	WCDataSelector,
 	PluginsStoreActions,
```

Comment:
@moon0326 Just a tiny thing. Could you remove unused `PluginsStoreActions`?

Reply:
@chihsuan updated in https://github.com/woocommerce/woocommerce/pull/32678/commits/32db78a76f31179621030c2add0b2d894965bfe3